schema-dex-amm-extended.graphql

```diff
@@ -721,14 +989,38 @@ type Position @entity {
   " Timestamp when the position was closed (0 if still open) "
   timestampClosed: BigInt
 
-  " Input token balances in this position, in native amounts "
-  inputTokenBalances: [BigInt!]!
+  ## lower tick of the position
+  tickLower: Tick
 
-  " Output token balance in this position, in native amounts "
-  outputTokenBalance: BigInt
+  ## upper tick of the position
+  tickUpper: Tick
 
-  " Total reward token accumulated under this position, in native amounts "
-  cumulativeRewardTokenAmounts: [BigInt!]
+  " Token that is to represent ownership of liquidity "
+  liquidityToken: Token
+
+  " Type of token used to track liquidity "
+  liquidityTokenType: TokenType
+
+  ## total position liquidity
+  liquidity: BigInt!
+
+  ## total position liquidity in USD
+  liquidityUSD: BigDecimal!
+
+  ## amount of tokens ever deposited to position
+  cumulativeDepositedTokenAmounts: [BigInt!]!
+
+  ## amount of tokens in USD deposited to position
+  cumulativeDepositedUSD: BigDecimal!
+
+  ## amount of tokens ever withdrawn from position (without fees)
+  cumulativeWithdrawnTokenAmounts: [BigInt!]!
```

Comment:
We thought about this naming in lending and named things like `cumulativeWithdrawTokenAmounts` bc we wanted to stick with the actual type name `Withdraw` not `Withdrawn`. I guess same goes for Deposited here as well

Reply:
Sounds good. Updated both to fit that convention.